viadot/sources/genesys.py

```diff
@@ -190,6 +194,9 @@ async def generate_post():
         coroutine = generate_post()
         loop.run_until_complete(coroutine)
 
+        if end_point == "conversations/details/query":
+            return json.loads(new_report.decode("utf-8"))
```

Comment:
The function returns 'dict' only if `endpoint = conversations/details/query`, right? What if the `end_point` is different?

Reply:
Returns None, that's why I use the typing `Union[None, dict]`. is that why you are asking for it?
I updated the doc-string.